src/Core/Domain/CustomerService/Command/UpdateContactOptionsCommand.php

```diff
@@ -0,0 +1,68 @@
+<?php
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+declare(strict_types=1);
+
+namespace PrestaShop\PrestaShop\Core\Domain\CustomerService\Command;
+
+class UpdateContactOptionsCommand
```

Comment:
It's interesting that you used a CQRS command for this one, however this form is more a configuration form than a CRUD one (for an entity). So out of consistency with the rest of the software it should follow the same architecture as other forms described here https://devdocs.prestashop-project.org/8/development/architecture/migration-guide/forms/settings-forms/

The doc may not be totally up to date (especially regarding the twig template which should be much simpler) but you can inspire from a existing configuration form like https://github.com/PrestaShop/PrestaShop/blob/21f6c963dd152bdff0aaed9c2c5399df899d6f48/src/PrestaShopBundle/Controller/Admin/Configure/ShopParameters/CustomerPreferencesController.php#L49